docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        circuits = [create_loschmidt_echo_circuit(qubits, cycles=c) for c in cycle_values]
</pre><p>Please add some more description about this code block. Why are again creating new loschmidt echo circuits instead of reusing the circuits we actually ran to get the probabilities? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Outdated, Loschmidt echo removed. (I only kept the last batch of echo circuits around, so here needed to create new ones on the fly. The alternative is to store all + reuse as you note.)</p>